docs/source/index.rst

```diff
@@ -75,6 +77,12 @@ or install it using your system package manager, e.g.::
 
     sudo apt-get install python3-pyqt5
 
+or::
+
+    sudo apt-get install python-pyqt
+    sudo apt-get install python3-pyqt
+
+if PyQt4 is desired.
```

Comment:
I'm not sure it's necessary to illustrate this; there's no particular reason the end user should care about using Qt 4 vs Qt 5. If they're using the Qt console as part of another Qt 4 application, the instructions for that application should get them set up.

If we do want to show it, then the names are `python-qt4` and `python3-pyqt4`.